iOS-sample-app/Showcases/Privacy/PrivacyViewController.swift

```diff
@@ -0,0 +1,63 @@
+import UIKit
+import ContentSquare
+
+// This class showcases how to use privacy-related APIs
+class PrivacyViewController: UIViewController {
+
+    @IBOutlet private weak var commandTitle: UILabel!
+    @IBOutlet private weak var commandMessage: UILabel!
+
+    override func viewWillAppear(_ animated: Bool)
+    {
+        super.viewWillAppear(animated)
+
+        commandTitle.text = ""
+        commandMessage.text = ""
+    }
+
+    //MARK: - UI Actions
```

Comment:
Could you add here a comment about the fact that the client has to ask himself for the user's consent, and then call our functions? Clients have to understand that we do not provide any UI to ask for the consent, it is their duty to do the implementation. We had in the past clients who were confused because they thought we were also providing it, and it only creates unuseful back and forth between them and us, so better just put a disclamer here - there should be one in our public documentation as well.